### DISCUSSION TITLE
Option for `quick restart` shortcut customization

### DISCUSSION THREAD

**Discussion by hibatillah** [monkeytypegame/monkeytype#4753] [Ideas] [answered]
Due to the frequent repetition of the typing process, my left little finger must always press the `tab` key to reset, and this is sometimes tiring (although this could be due to excessive use).

It would be better if you could customize the `quick restart` button, so that it can be adjusted to each user's preferences, or to avoid tired left little fingers like mine.

Similar things have also been discussed in this #3549

Hopefully developers can help to add this feature soon...
thank you🙌

**Comment by Miodec** [comment, marked as answer]
I can add an "enter" option sure, but im not sure about making it completely user selectable. I mean, what else would you set it to?

**Reply by SnarkyDeveloper** [reply]
I would like to see / as an option, since its easy to hit

**Comment by hibatillah** [comment]
I actually didn't think too much about that either😅, the addition of `enter` option has been a really good help in my opinion.
Hopefully soon, thank you for your help👌
